src/main/java/io/strimzi/kafka/bridge/http/HttpConfig.java

```diff
@@ -20,10 +20,12 @@
     public static final String HTTP_ENABLED = HTTP_CONFIG_PREFIX + "enabled";
     public static final String HTTP_HOST = HTTP_CONFIG_PREFIX + "host";
     public static final String HTTP_PORT = HTTP_CONFIG_PREFIX + "port";
+    public static final String HTTP_CONSUMER_TIMEOUT = HTTP_CONFIG_PREFIX + "timeout";
 
     public static final boolean DEFAULT_HTTP_ENABLED = true;
     public static final String DEFAULT_HOST = "0.0.0.0";
     public static final int DEFAULT_PORT = 8080;
+    public static final long DEFAULT_CONSUMER_TIMEOUT = 60 * 1000 * 1000;
```

Comment:
Still in doubt about having 1 hour as default timeout ... @tombentley wdyt?

Reply:
Regarding the good point about disabling the timeout.
The way that makes more sense to me is just not having the `http.timeout` property in the configuration. Actually, it doesn't work, because not putting a property means using the default value which is how all the other parameters work, so I would leave in that way.
The solutions could be just relying on the user, documenting it, that for "disabling" he should se a pretty high value there or using a special value as 0 (some tools/library disable timing related features in this way). Wdyt? @stanlyDoge @tombentley

Reply:
I think -1 is more obviously a special value than 0 (even though 0 doesn't have a sensible meaning).

Also @stanlyDoge as a matter of habit when setting a `long`-typed variable to a result of a multiplication like this it is always a good idea to say `60L * 1000L * 1000L` so the expression is evaluated using longs. Mostly you get away with it, but that makes those occasions where the result overflows `int` atrithmetic and is then widened into a `long` all the more confusing.

Reply:
I guess `-1` makes more sense. You can also add another boolean option which would be true by default and would be enabling / disabling the timeout.

Reply:
Now you provided an opinion on the suggested values but then adding a new option :-) Let's go with -1 as this is ok for you and Tom. @stanlyDoge now you have a quorum!